src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -108,18 +108,8 @@ const Packages = ({ defaultArch, ...props }) => {
   };
 
   const getAllPackages = async () => {
-    const args = [
-      getState()?.values?.release,
-      getState()?.values?.architecture || defaultArch,
-      packagesSearchName,
-    ];
-    let { data, meta } = await api.getPackages(...args);
-    if (data?.length === meta.count) {
-      return data;
-    } else if (data) {
-      ({ data } = await api.getPackages(...args, meta.count));
-      return data;
-    }
+    const args = [getState()?.values?.release, packagesSearchName];
+    return await api.getPackages(...args);
```

Comment:
I think we should only use the content-service on stage-beta for now; which can be checked against the insights.chrome object (isBeta && !isProd).

Reply:
Not the cleanest since the data responses, and therefore the handling, is different. But, it should differentiate where to call now.